scripts/preptest.sh

```diff
@@ -1,12 +1,12 @@
-#!/bin/bash
+#!/usr/bin/env bash
 #
 # Prep a base Drupal install for memcache testing.
 # This assumes that $WEBDIR contains a working Drupal install.
 
 WEBDIR=/var/www/html
 MEMCACHE_SETTINGS_FILE=memcache.settings.inc
-MEMCACHE_VERSION=7.x-1.2
-DATABASE_NAME=drupal
+MEMCACHE_VERSION=6.x-1.11
```

Comment:
Worth having two versions of this script or accept command line args for 6 vs. 7?